packages/itwinui-react/src/core/Menu/Menu.tsx

```diff
@@ -89,9 +94,9 @@ export const Menu = React.forwardRef((props, ref) => {
       as='div'
       className={cx('iui-menu', className)}
       role='menu'
-      onKeyDown={onKeyDown}
       ref={refs}
       {...rest}
+      onKeyDown={mergeEventHandlers(props.onKeyDown, onKeyDown)}
```

Comment:
nit: Reorder to reduce diff

```suggestion
      role='menu'
      onKeyDown={mergeEventHandlers(props.onKeyDown, onKeyDown)}
      ref={refs}
      {...rest}
```

Reply:
no, the order is intentional. `...rest` would override `props.onKeyDown`